app/controllers/people_controller.rb

```diff
@@ -147,6 +115,15 @@ def destroy
   end
 
   private
+
+  def manage_people_depts
+    @available_departments = Department.where(manage_people: true)
```

Comment:
I think we'd need `Department.active.where....`

Reply:
@kgf I can change this in the commit to change the checkbox filter rendering.